jib-core/src/test/java/com/google/cloud/tools/jib/docker/DockerLoadManifestBlobTest.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2018 Google LLC. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.google.cloud.tools.jib.docker;
+
+import com.google.cloud.tools.jib.blob.Blob;
+import com.google.cloud.tools.jib.image.ImageReference;
+import com.google.common.io.Resources;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Arrays;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.Test;
+
+/** Tests for {@link DockerLoadManifestBlob}. */
+public class DockerLoadManifestBlobTest {
+
+  @Test
+  public void testGetDockerLoadManifest() throws URISyntaxException, IOException {
+    // Loads the expected JSON string.
+    Path jsonFile = Paths.get(Resources.getResource("json/loadmanifest.json").toURI());
+    String expectedJson = new String(Files.readAllBytes(jsonFile), StandardCharsets.UTF_8);
+
+    List<String> layers = Arrays.asList("layer1.tar.gz", "layer2.tar.gz", "layer3.tar.gz");
+    ImageReference testImage = ImageReference.of(null, "testrepo", "testtag");
+    Blob manifestBlob = DockerLoadManifestBlob.get(testImage, layers);
+    try (ByteArrayOutputStream jsonStream = new ByteArrayOutputStream()) {
+      manifestBlob.writeTo(jsonStream);
```

Comment:
`Blobs#writeToString` would give the string directly.

Reply:
It's kind of confusing to have a `writeTo` method in the Blob interface, but then have a bunch of other `writeTo` methods in the Blobs class, would it make sense to combine these somehow?

Reply:
So the design for that is similar to classes like Files vs. Path, where the plural is a bunch of static methods that operate on the singular. `Blob` is an interface that is implemented with different ways it is written out to an `OutputStream`, whereas `Blobs` is a bunch of final static methods for working with `Blob`s. The methods are prob better named `toString` and `toByteArray`.

Reply:
Files and Path are also pretty confusing.

Reply:
I meant more in the sense that you have functionality for one system split into two different classes, using two different styles. What if `Blob` was abstract and contained the methods currently in `Blobs`? I know abstract classes seem to be avoided in favor of interfaces but I'm not a fan of splitting things out like this.

Reply:
Putting the methods in `Blobs` into an abstract `Blob` would not actually work since it introduces circular dependencies where a parent class (`Blob`) depends on implementation classes (like `InputStreamBlob`).